core/store/migrations/migration1592355365/migrate.go

```diff
@@ -0,0 +1,22 @@
+package migration1592355365
+
+import (
+	"github.com/jinzhu/gorm"
+)
+
+// Migrate ensures that heads are unique and adds parent hash for use in reorg detection
+func Migrate(tx *gorm.DB) error {
+	return tx.Exec(`
+		CREATE TABLE "job_spec_errors" (
+			"id" uuid primary key NOT NULL,
```

Comment:
Why is this a UUID?

Reply:
at first it was necessary to work with delete requests on the frontend, but that has since been updated.  Converting back to an int.